src/components/searchbox/SearchBox.tsx

```diff
@@ -0,0 +1,37 @@
+import styled from "styled-components";
+
+type SearchBoxProps = {
+  searchChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+};
+
+const SearchBox = ({ searchChange }: SearchBoxProps) => {
```

Comment:
There is a duplicate component `SearchBox.tsx` in the project. The unused component should be removed.

Reply:
I deleted the other one searchbox